BarrysPeanutsAkkaCascading/src/main/java/barryspeanuts/actor/CheckOutActor.java

```diff
@@ -0,0 +1,71 @@
+package barryspeanuts.actor;
+
+import akka.actor.typed.ActorSystem;
+import akka.actor.typed.Behavior;
+import akka.actor.typed.javadsl.AbstractBehavior;
+import akka.actor.typed.javadsl.ActorContext;
+import akka.actor.typed.javadsl.Behaviors;
+import akka.actor.typed.javadsl.Receive;
+import barryspeanuts.msg.CreditCard;
+import barryspeanuts.msg.Customer;
+import barryspeanuts.msg.PurchaseItem;
+import java.util.ArrayList;
+import org.slf4j.Logger;
+import org.slf4j.LoggerFactory;
+
+public class CheckOutActor extends AbstractBehavior<Object> {
+  Logger logger = LoggerFactory.getLogger(ShoppingCartActor.class);
+
+  private CheckOutActor(ActorContext<Object> context) {
+    super(context);
+  }
+
+  public static Behavior<Object> create() {
+    return Behaviors.setup(CheckOutActor::new);
+  }
+
+  public static Behavior<Object> behavior() {
+    return Behaviors.setup(CheckOutActor::new);
+  }
+
+  @Override
+  public Receive<Object> createReceive() {
+    return newReceiveBuilder().onMessage(StartCheckout.class, this::handleStartCheckout).build();
+  }
+
+  private Behavior<Object> handleStartCheckout(StartCheckout msg) {
+    logger.info("{} is Checking Out a purchase and preparing a payment", CheckOutActor.class);
```

Comment:
```suggestion
    logger.info("{} is Checking Out a purchase and preparing a payment", msg.getCustomer());
```

Giving the class type doesn't make sense (and the logger is already created with that)

Reply:
The cascading code has been removed from the project, @cretz .

Reply:
:+1: This comment was from weeks ago. You can mark these issues as resolved if they are resolved.